drivers/hmis_csv_importer/app/models/hmis_csv_importer/import_override.rb

```diff
@@ -125,6 +130,10 @@ def describe_when
     "#{replaces_column} is #{replaces_value}" if replaces_value.present?
   end
 
+  def describe_why
+    description == ':NULL:' ? nil : description
```

Comment:
Why does the description need the `:NULL:`?  It shouldn't affect the processing at all, I think this method can just go away and the table can just call `override.description`

Reply:
I can adjust this one, I had it set this way to mimic the way the other fields were being displayed.

Reply:
Yeah, I don't think we need to yet.  Thanks!

Reply:
This has been updated - https://github.com/greenriver/hmis-warehouse/pull/4791/commits/bcb01769041fb8718afddcb7488859d6a81f4c8d